spec/specification/json_spec.rb

```diff
@@ -16,6 +16,7 @@ module Pod
             'tvos' => nil,
             'watchos' => nil,
           },
+          'test_specification' => false,
```

Comment:
sorry I dont quite get the request here, are you implying we should write `nil` or not write this value at all and just treat the fact that its missing as `false`and write it only when its `true`?